core/src/main/java/org/apache/druid/java/util/common/HumanReadableBytes.java

```diff
@@ -227,7 +235,12 @@ private static long parseInner(String rawNumber)
      * simplified SI format without 'B' indicator
      * eg: K, M, G ...
      */
-    DECIMAL
+    DECIMAL,
+    /**
+     * same as binary byte format, that k8s uses
+     * eg: Ki, Mi, Gi ...
+     */
+    KUBERNETES_BYTE
```

Comment:
These enums are designed for the format function `HUMAN_READABLE_XXXX_FORMAT` on SQL. At the SQL layer, since you don't add a new SQL function, I think we don't need to add format support here. Another reason is that I think at the SQL layer, it's better keeping the standard format function only.

Reply:
yeah sure, ill remove this and corresponding tests.  I didn't know if it was used anywhere else from a config standpoint to do any conversion.  But if it is only for SQL better to remove it.